sunpy/map/compositemap.py

```diff
@@ -342,7 +341,8 @@ def draw_grid(self, index=None, axes=None, grid_spacing: u.deg = 20*u.deg, **kwa
 
     def plot(self, axes=None, annotate=True,
              title="SunPy Composite Plot", **matplot_args):
-        """Plots the composite map object using matplotlib
+        """Plots the composite map object by calling :meth:`~sunpy.map.GenericMap.plot`
+        or :meth:`~sunpy.map.GenericMap.draw_contours`.
```

Comment:
What actually decides whether the map is plotted or contoured?

Reply:
If I remember correctly you set it with a different part of the composite map API, i.e. set the type for each map in the collection.

Reply:
If `.levels` is set for a given map, that map will be contours instead of an images

Reply:
Added more to the docstring